src/components/Home.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+import ArticleSearch from "./ArticleSearch";
+import ArticleList from "./ArticleList";
+
+function Home() {
+  const [articles, setArticles] = useState([]);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+
+  return (
+    <>
+      <ArticleSearch
+        setArticles={setArticles}
+        setIsError={setIsError}
+        setIsLoading={setIsLoading}
+      />
+      <ArticleList
+        articles={articles}
+        isError={isError}
+        isLoading={isLoading}
+      />
+    </>
```

Comment:
I would actually say that `Home` would be a great place to have the useEffect to fetch `articles` and pass them to the list to be displayed 💡